nemo_aligner/algorithms/ppo.py

```diff
@@ -28,8 +28,10 @@
 from nemo.collections.nlp.data.language_modeling.megatron.data_samplers import MegatronPretrainingRandomSampler
 from nemo.collections.nlp.modules.common.megatron.utils import get_iterator_k_split
 from nemo.utils import logging
+from nemo_aligner.models.nlp.gpt.megatron_gpt_ppo_actor import MegatronGPTActorModel
 from nemo_aligner.utils import parallel_state
 from nemo_aligner.utils.distributed import (
+    ScopedTimer,
     SyncTimer,
```

Comment:
seems unused

Reply:
rm in 63cb223e051ddd9609ad426082883e4aca7a4100